src/interface/client.js

```diff
@@ -34,9 +30,13 @@ export function request(options) {
 
             options.window = el.contentWindow;
 
-            if (!options.window) {
-                throw new Error('Expected options.window');
-            }
+            
+        } else if (options.window.tagName && options.window.tagName.toLowerCase() === 'iframe') {
```

Comment:
If `options.window` is a cross-domain window, checking `options.window.tagName` is going to throw a cross-domain error.